<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Patcher</class>
 <widget class="QMainWindow" name="Patcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MHP3rd patcher v1.4</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>267</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>52</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ISO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="iso_path">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="iso_search">
         <property name="minimumSize">
          <size>
           <width>78</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>52</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Parche</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="patch_list">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="msg_list">
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="iso_patch">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Parchear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionBuscar"/>
    <addaction name="actionParchear"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAcerca_de_Qt"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionAcerca_de_Qt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="text">
    <string>Buscar</string>
   </property>
  </action>
  <action name="actionParchear">
   <property name="text">
    <string>Parchear</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>Patcher</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
